File: src/gui/MainMenuController.java
package gui;

import com.jfoenix.controls.JFXButton;
import javafx.fxml.FXML;

public class MainMenuController {

    @FXML
    private JFXButton startButton;

    @FXML
    private JFXButton quitButton;

    public JFXButton getStartButton() {
        return startButton;
    }

    public JFXButton getQuitButton() {
        return quitButton;
    }

    @FXML
    public void initialize() {
        assert startButton != null : "fx:id=\"startButton\" was null check your FXML ";
        assert quitButton != null : "fx:id=\"quitButton\" was null check your FXML ";
    }

}

File: src/gui/MainMenuController.kt
package gui

import com.jfoenix.controls.JFXButton
import javafx.fxml.FXML

class MainMenuController {

    @FXML
    var startButton: JFXButton? = null

    @FXML
    var quitButton: JFXButton? = null

    fun initialize() {
        assert(startButton != null) { "fx:id=\"startButton\" was null check your FXML " }
        assert(quitButton != null) { "fx:id=\"quitButton\" was null check your FXML " }
    }

}
